Makefile

```diff
@@ -1,8 +1,17 @@
 SHELL := /bin/bash
 
-run:
+run: create-db-user
```

Comment:
This will try to create the user before creating the infrastructure. Could you make sure to test this while setting up a new project?